generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/DateTime/TemporalFieldValueSource.java

```diff
@@ -18,6 +19,7 @@
     public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999_999_999);
     public static final LocalDateTime ISO_MIN_DATE = LocalDateTime.of(1, 1, 1, 0, 0);
 
+    private final ChronoUnit granularity = ChronoUnit.MILLIS;
```

Comment:
field should be `static` too, it never changes

Reply:
it won't be static in the future, i don't see much point in making it static now